### DISCUSSION TITLE
My profile won't show in the homepage

### DISCUSSION THREAD

**Discussion by macenteno63** [nunocoracao/blowfish#1807] [Q&A] [answered]
here is my params.toml
![Capture d’écran du 2024-10-21 15-54-08](https://github.com/user-attachments/assets/c78f4b1b-4746-4ad3-8319-0e9f073e019b)

my config.toml : 

![Capture d’écran du 2024-10-21 15-56-00](https://github.com/user-attachments/assets/d861f968-9146-4df2-8792-0d18bb9dd0ac)

the result : 

![Capture d’écran du 2024-10-21 15-56-47](https://github.com/user-attachments/assets/30df3ec2-c2f5-452b-a478-c7544dc01a1a)

my author information don't appear...

I also have this warning : 
![Capture d’écran du 2024-10-21 15-58-48](https://github.com/user-attachments/assets/64582d49-667c-4893-b004-339dd975c2e3)

Can someone help me please.

github repo for full code : https://github.com/macenteno63/portfolio

**Comment by dosoft** [comment, marked as answer]
@macenteno63 Use params.author or move [author] section to the params.toml
